ml/src/test/scala/ai/h2o/sparkling/ml/models/H2OSupervisedMOJOModelTestSuite.scala

```diff
@@ -43,7 +42,7 @@ class H2OSupervisedMOJOModelTestSuite extends FunSuite with Matchers with Shared
   private lazy val Array(trainingDataset, testingDataset) = dataset.randomSplit(Array(0.8, 0.2), 1234L).map(_.cache())
 
   def testOffsetColumnGetsPropagatedToMOJOModel(
-      algo: H2OSupervisedAlgorithm[_ <: H2OBaseModelBuilder, _ <: H2OBaseModel, _ <: Model.Parameters]): Unit = {
+                                                 algo: H2OSupervisedAlgorithm[_ <: Model.Parameters]): Unit = {
```

Comment:
nit: we could have method definition at one line

Reply:
👍